test/functional/ownership_requests_controller_test.rb

```diff
@@ -244,6 +244,122 @@ class OwnershipRequestsControllerTest < ActionController::TestCase
         end
       end
     end
+
+    context "when user owns a gem with more than MFA_REQUIRED_THRESHOLD downloads" do
+      setup do
+        @mfa_rubygem = create(:rubygem)
+        create(:ownership, rubygem: @mfa_rubygem, user: @user)
+        GemDownload.increment(
+          Rubygem::MFA_REQUIRED_THRESHOLD + 1,
+          rubygem_id: @mfa_rubygem.id
+        )
+        @request.cookies[:mfa_required] = "true"
+        @rubygem = create(:rubygem)
+        create(:ownership_call, rubygem: @rubygem)
+        @ownership_request = create(:ownership_request)
+      end
+
+      context "user has mfa disabled" do
+        context "POST to create" do
+          setup { post :create, params: { rubygem_id: @rubygem.name, note: "small note" } }
+
+          should redirect_to("the setup mfa page") { new_multifactor_auth_path }
+        end
+
+        context "PATCH to close_all" do
+          setup { patch :close_all, params: { rubygem_id: @rubygem.name } }
+
+          should redirect_to("the setup mfa page") { new_multifactor_auth_path }
+        end
```

Comment:
nit: could use the same pattern used in https://github.com/rubygems/rubygems.org/pull/3153/files#diff-9c8ad0e0aeaea2d49104622aacfdae9948cbc2d6be643dea1bc5a7cbef9b7b34R310-R318

Reply:
I think for having just two tests, this is clearer, albeit less consistent